.github/labeler.yml

```diff
@@ -0,0 +1,273 @@
+### labeler.yaml
+### This is a best naive guess on auto labeling
+### with this script: https://repl.it/@maxandersen/labels
+### should be verified/tuned by changing the script and once
+### committed updates would be done manually.
+
+
+### Non-covered dirs....
+### - extensions
+### - extensions/agroal
+### - extensions/amazon-dynamodb
+### - extensions/azure-functions-http
+### - extensions/caffeine
+### - extensions/flyway
+### - extensions/jackson
+### - extensions/jdbc
+### - extensions/jgit
+### - extensions/jsch
+### - extensions/jsonb
+### - extensions/jsonp
+### - extensions/keycloak-authorization
+### - extensions/mailer
+### - extensions/netty
+### - extensions/pom.xml
+### - extensions/quartz
+### - integration-tests
+### - integration-tests/.project
+### - integration-tests/.settings
+### - integration-tests/amazon-dynamodb
+### - integration-tests/class-transformer
+### - integration-tests/flyway
+### - integration-tests/jackson
+### - integration-tests/jgit
+### - integration-tests/jsch
+### - integration-tests/jsonb
+### - integration-tests/keycloak
+### - integration-tests/keycloak-authorization
+### - integration-tests/main
+### - integration-tests/pom.xml
+### - integration-tests/quartz
+### - integration-tests/shared-library
+### - integration-tests/test-extension
+### - integration-tests/virtual-http
+
+### Non-covered area/
+### area/bootstrap-maven
+### area/build
+### area/camel
+### area/core
+### area/dependencies
+### area/dev-mode
+### area/gradle
+### area/housekeeping
+### area/infra
+### area/new-http
+### area/oidc-interoperability
+### area/openapi
+### area/platform
+### area/reactive-messaging
+### area/security-consolidation
+### area/testing
+### area/user experience
+area/amazon-lambda:
+ - extensions/amazon-lambda/**/*
+ - extensions/amazon-lambda-http/**/*
+ - integration-tests/amazon-lambda/**/*
+ - integration-tests/amazon-lambda-http/**/*
+ - integration-tests/amazon-lambda-http-it/**/*
+ - integration-tests/amazon-lambda-http-resteasy/**/*
+
+area/arc:
+ - extensions/arc/**/*
+
+area/artemis:
+ - extensions/artemis-core/**/*
+ - extensions/artemis-jms/**/*
+ - integration-tests/artemis-core/**/*
+ - integration-tests/artemis-jms/**/*
+
+area/config:
+ - extensions/config-yaml/**/*
+
+area/documentation:
+ - docs/**/*
+
+area/hibernate-orm:
+ - extensions/hibernate-orm/**/*
+ - integration-tests/hibernate-orm-panache/**/*
+
+area/hibernate-search:
+ - extensions/hibernate-search-elasticsearch/**/*
+ - integration-tests/hibernate-search-elasticsearch/**/*
+
+area/hibernate-validator:
+ - extensions/hibernate-validator/**/*
+ - integration-tests/hibernate-validator/**/*
+
+area/infinispan:
+ - extensions/infinispan-client/**/*
+ - extensions/infinispan-embedded/**/*
+ - integration-tests/infinispan-cache-jpa/**/*
```

Comment:
You can add this one to Hibernate too. It needs both labels IMHO.

Reply:
i'll probably write a new script to generate this stuff automatically ;) 
so can we say any folder with `jpa` in it should get the hibernate label ?